### DISCUSSION TITLE
What's the difference between components in UI package and Presentation Package.

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#3295] [Q&A] [answered]
I found that there are many components of the same name across the UI package and Presentation package, such as PropertyGrid, Table, but with difference state and props type. 
I'm confused about which to choose. Is Presentation Package for certain circumstance？

**Comment by grigasp** [comment, marked as answer]
UI packages provide pure UI components which can be used to display any kind of data.

The `presentation-components` package provides various APIs that help you feed [presentation data](https://www.itwinjs.org/presentation/) into those UI components. You should use this package if you're planning to use presentation rules for creating data that you're going to display in your UI.
